src/rp2/locales/es/LC_MESSAGES/messages.po

```diff
@@ -631,6 +630,46 @@ msgstr "Resumen de ganancia / pérdida anual"
 msgid "Yellow"
 msgstr "Amarillo"
 
+msgid "airdrop"
+msgstr ""
```

Comment:
Missing translation: is this intended?

Reply:
intended, no good translation in spanish.